.github/workflows/ci.yaml

```diff
@@ -0,0 +1,176 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - "**"
+    tags:
+      - "[0-9]+.[0-9]+.[0-9]+**"
+
+env:
+  GO_VERSION: 1.16
+  GOFLAGS: -mod=readonly
+
+jobs:
+  # Note: currently jobs.<job-id>.(runs-on|strategy) cannot use global '${{ env }}'.
+  #
+  # Note: currently ${{ env }} cannot define dynamically initialized values.
+  environment:
+    name: Configure environment
+    runs-on: ubuntu-latest # TODO: until static global env can be used through ${{ env }}.
+    outputs:
+      git_refname: ${{ steps.set-git-refname.outputs.git_refname }}
+      github_runner_default_os: ubuntu-latest
+
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v2
+
+      - name: Set Git refname
+        id: set-git-refname
+        run: echo ::set-output name=git_refname::$(echo "${{ github.ref }}" | sed -r 's@refs/(heads|pull|tags)/@@g' )
```

Comment:
We use `git_refname` in a single place and `github_runner_default_os` everywhere instead of a hardcoded value. Is it really worth the extra step and the extra level of indirection? IMHO it's quite easy to update `ubuntu-latest` everywhere in a file if we have to (not that it really matters at the moment).

Reply:
Fixed in c31fbd3.